generatebundlefile/bundle.go

```diff
@@ -65,10 +65,13 @@ func (projects Project) NewPackageFromInput() (*api.BundlePackage, error) {
 	if err != nil {
 		return nil, err
 	}
-	versionList, err := ecrClient.GetShaForTags(projects)
+	versionList, err := ecrClient.GetShaForInputs(projects)
 	if err != nil {
 		return nil, err
 	}
+	if len(versionList) < 1 {
+		return nil, fmt.Errorf("error: Unable to find SHA sum for given input tag %v", projects.Versions)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to find SHA sum for given input tag: %v", projects.Versions)
```
No need to prefix an error with "error".